staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -311,6 +327,15 @@ func InstrumentHandlerFunc(verb, group, version, resource, subresource, scope, c
 	}
 }
 
+// cleanContentType binds the contentType (for metrics related purposes) to a
+// bounded set of known/expected content-types.
+func cleanContentType(contentType string) string {
+	if validMetricContentTypes.Has(contentType) {
+		return contentType
+	}
+	return OtherContentType
```

Comment:
If any of the metric types are unknown, should we log the true value, so we know if there's a type that should be added to the list? Same to other labels.

Reply:
All requests are already logged in audit logs, so we shouldn't be losing any data.

Reply:
I don't think we record content type in the audit logs though?

Reply:
If it's not there, then I'm pretty sure it's in the kube-apiserver logs when verbosity=3 (enabling request logging).